### DISCUSSION TITLE
Choosing which badges to display

### DISCUSSION THREAD

**Discussion by piazzai** [community/community#20473] [Other Feature Feedback, Questions, & Ideas]
I dislike the new badges. I find them childish and demeaning. I liked the old badges (Arctic, Sponsor, Mars), and I would like to display them on my profile and keep seeing them on others' profiles.

Whenever someone points out they don't like the badges, GitHub staff replies: You can turn them off in your profile settings. I know that, but (1) you turn off all of them or none of them, and I want to keep seeing the Arctic, Sponsor and Mars badges; (2) you only turn them off on your profile, but keep seeing sharks, pea pods, and other nonsense on other people's.

A switch to turn off the new badges across the entire website would be most welcome.

**Comment by radar-blip** [comment]
Maybe there will be a feature to do that one day. They may even be working on it right now. Only time will tell. 👍🏼
